sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/key_client_models.hpp

```diff
@@ -130,6 +131,12 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
      * CancellationToken) method.
      */
     AZ_SECURITY_KEYVAULT_KEYS_DLLEXPORT static const KeyOperation Import;
+
+    /**
+     * @brief The key can be exported during creation using the ImportKey(ImportKeyOptions,
+     * CancellationToken) method.
```

Comment:
Same issue here and elsewhere in this file:
https://github.com/Azure/azure-sdk-for-cpp/blob/2feed76d860eaa5ac4697f6a235337a54285154d/sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/key_client_models.hpp#L92-L132